src/app/file-browser/components/sidebar/sidebar.component.ts

```diff
@@ -168,6 +168,12 @@ export class SidebarComponent implements OnDestroy, HasSubscriptions {
     }
   }
 
+  onLocationEnterPress(e: KeyboardEvent): void {
+    if (this.canEdit && e.key === 'Enter') {
+      this.editService.openLocationDialog(this.selectedItem);
+    }
+  }
```

Comment:
Can you cover this function with some basic tests? You do not need to cover the rest of the component for now.